### ISSUE TITLE
Rounded corners (of PLI) are not round but oval

### ISSUE THREAD

**Issue by Landcross** [trevorsandy/lpub3d#473]
I think the images speak for themselves:
![image](https://user-images.githubusercontent.com/12005521/97497259-85fb5180-196a-11eb-90a2-f10024bfd6fd.png)

![image](https://user-images.githubusercontent.com/12005521/97497226-78de6280-196a-11eb-9568-8fd611eb7a10.png)

I expect a normal rounded corned that has the same radius on both ends, but instead I'm getting this weird oval shape. To be clear, I expect something like this:
![image](https://user-images.githubusercontent.com/12005521/97497437-c529a280-196a-11eb-8e1a-b126e569330d.png)

I'm using the latest LPub3D 2.4.0, though I'm not sure if this issue existed before or not.

**Comment by Landcross** [comment]
Small update: if the PLI is square(ish), the rounded corners look fine:
![image](https://user-images.githubusercontent.com/12005521/97561947-32781a80-19e1-11eb-85d2-358875f74fc8.png)

So there seems to be a discrepancy in the calculation in in regards to top/bottom and left/right.

**Comment by trevorsandy** [comment]
Thank you for reporting this behaviour. Indeed, there seems to be a discrepancy in the radius calc. I'll take a look.

Cheers,

**Comment by Landcross** [comment]
I just installed 2.3.13 again (because of the various issues I thus far reported and I need to finish instructions soon) and I loaded up the file with which I reported this issue and I noticed that all PLI border settings I made before with 2.4 weren't taking effect in 2.3.13.

I'm not sure why LPub 2.3 doesn't use the PLI border settings I created with 2.4, but they are as follows. The only difference I could find in the meta commands is that 2.3 creates a border radius of 100 but 2.4 created a radius of 100.000. Not sure if that matters.
